src/type-generator.ts

```diff
@@ -111,8 +136,32 @@ export class TypeGenerator {
     });
   }
 
+  private buildEnumType(enumType: pgStructure.Type) {
+    const types = (enumType as any).values.map((t: string) => `'${t}'`).join(
```

Comment:
Personally I'd defer this join to the function below, and return an array here.

Reason being, if we want to ever switch from unions to enums you'll only need to edit one place, with current design you need to edit two.

Or to put it differently keep data in the most primitive form as long as you can, and then just render it.